examples/generate_malicious_tarball.rs

```diff
@@ -0,0 +1,40 @@
+extern crate tar;
```

Comment:
Perhaps this script could just be part of the test itself? It seems odd to have an example which generates a test case.